docs/builtin_elements.md

```diff
@@ -17,6 +17,9 @@ These properties are valid on all visible items
   children with transparency. 0 is fully transparent (invisible), and 1 is fully opaque. (default: 1)
 * **`visible`** (*bool*): When set to `false`, the element and all his children will not be drawn
   and not react to mouse input (default: `true`)
+* **`layer`** (*bool*): When set to `true`, this provides a hint to the renderer to cache the contents of the element and all the children into an intermediate cached layer.
```

Comment:
Perhaps this should be called `layer-hint` instead of `layer`.

Reply:
`cache: true;` ?  `cache-hint: true;`  ?

Reply:
`cache` is a bit too generic IMO. Maybe `rendering-cache-hint`?